docs/directives.md

```diff
@@ -0,0 +1,55 @@
+# @glint Directives
+
+Glint has several directives that can be used in template comments to control
+Glint's behavior. Additional comment text can follow directives to document
+their purpose.
```

Comment:
Do you think it's worth calling out that these correspond to `@ts` directives of the same name to help with the mental model?

Reply:
Yes, I was actually adding that this morning!